src/components/calcite-slider/calcite-slider.tsx

```diff
@@ -41,57 +42,64 @@ export class CalciteSlider {
   /** Disable and gray out the slider */
   @Prop({ reflect: true }) disabled = false;
 
-  /** Minimum selectable value */
-  @Prop({ reflect: true }) min = 0;
+  /** Indicates if a histogram is present */
+  @Prop({ reflect: true, mutable: true }) hasHistogram = false;
+
+  /** Display a histogram above the slider */
+  @Prop() histogram?: DataSeries;
+
+  @Watch("histogram") histogramWatcher(newHistogram: DataSeries): void {
+    this.hasHistogram = !!newHistogram;
+  }
+
+  /** Label handles with their numeric value */
+  @Prop({ reflect: true }) labelHandles?: boolean;
+
+  /** Label tick marks with their numeric value. */
+  @Prop({ reflect: true }) labelTicks?: boolean;
```

Comment:
nitpick: Do we have guidelines for when boolean props should be initialized to false vs undefined?

For example, hasHistagram = false but labelTicks = undefined. mirrored = false;

Its not a big deal just a consistency thing :)

Reply:
I know we have the same inconsistencies with strings. Some are undefined by default and others are "";

Reply:
We don't have any guidelines at the moment for defaults. Whenever possible, I think we should initialize to a sensible value (similar to native elements). This use case is weird to me if we don't:

```ts
const slider = document.createElement("calcite-slider");
console.log(slider.labelTicks); // undefined even though the doc types it as boolean
```

There could be some exceptions. For example, if we treat `intl` strings as overrides instead of defaults we could skip initializing to keep the defaults internal and to also promote reuse between components (e.g., [`calcite-time-picker#intlHour`](https://github.com/Esri/calcite-components/blob/master/src/components/calcite-time-picker/calcite-time-picker.tsx#L59) vs. [`calcite-input-time-picker#intlHour`](https://github.com/Esri/calcite-components/blob/master/src/components/calcite-input-time-picker/calcite-input-time-picker.tsx#L49) )